src/lib/services/contentful/fetch.ts

```diff
@@ -0,0 +1,31 @@
+import { VERCEL, CONTENTFUL_SPACE_ID, CONTENTFUL_DELIVERY_API_TOKEN } from "$env/static/private";
+
+// Re-evaluate utils using env vars on each request to ensure tests can
+//   properly mock and overwrite them.
+const contentfulConnected = (): boolean =>
+  !!(VERCEL === "1" && CONTENTFUL_SPACE_ID && CONTENTFUL_DELIVERY_API_TOKEN);
+
+const graphApiUrl = (): string =>
+  `https://graphql.contentful.com/content/v1/spaces/${CONTENTFUL_SPACE_ID}`;
+
+const graphApiOptions = () => ({
+  method: "POST",
+  headers: {
+    "Content-Type": "application/json",
+    Authorization: `Bearer ${CONTENTFUL_DELIVERY_API_TOKEN}`,
+  },
+});
```

Comment:
Obviously I'd prefer to evaluate these values only once, instead of on every request. If you've got some ideas let me know!

Reply:
two thoughts:

1. I might be missing something obvious, but is there a reason these can't just be declared as objects instead of functions?
2. If (1) isn't an option, would this be a valid use of a [`readable` store](https://svelte.dev/docs#run-time-svelte-store-readable)?

Reply:
I think using objects would still give us trouble when testing; the readable store might work with a mock though. I had made one early attempt at using stores here (never committed), so I could give that another go. Will give that a try after I finish up some things with the nav in #132.

Reply:
I'm going to take a shot at representing these without functions, I think we can manage it in a similar way to how we mock the env variables.